src/lib/bootloader/grub_install.rb

```diff
@@ -72,10 +72,12 @@ def report_failure(exception)
     # creates basic command for grub2 install without specifying any stage1
     # locations
     def basic_cmd(secure_boot, trusted_boot)
-      if secure_boot
+      if secure_boot && !Yast::Arch.aarch64
         cmd = ["/usr/sbin/shim-install", "--config-file=/boot/grub2/grub.cfg"]
       else
         cmd = ["/usr/sbin/grub2-install", "--target=#{target}"]
+        # On aarch64, we do not use shim, but '--suse-signed-grub' option
```

Comment:
please reference there link to bugzilla entry like `(bsc#1136601)`. It will help in future for reasoning and more detailed info. Similar way how it was at https://github.com/yast/yast-bootloader/pull/565/files#diff-b0238c1d8aa9d0846b284a3b971bc543L64.

Reply:
Done